ground-ingest/src/main/java/edu/berkeley/ground/ingest/Test.java

```diff
@@ -0,0 +1,30 @@
+package edu.berkeley.ground.ingest;
+
+import java.io.IOException;
+import java.net.URI;
+
+import org.apache.hadoop.conf.Configuration;
+import org.apache.hadoop.fs.FileStatus;
+import org.apache.hadoop.fs.FileSystem;
+import org.apache.hadoop.fs.Path;
+
+public class Test {
+
+  public static void main(String[] args) throws IOException {
```

Comment:
Couple things:

1. What is this test testing?
2. Why is this in `src/main` and not `src/test`?